nixos/modules/system/boot/networkd.nix

```diff
@@ -94,7 +94,7 @@ let
   checkNetwork = checkUnitConfig "Network" [
     (assertOnlyFields [
       "Description" "DHCP" "DHCPServer" "IPForward" "IPMasquerade" "IPv4LL" "IPv4LLRoute"
-      "LLMNR" "MulticastDNS" "Domains" "Bond" "IPv6PrivacyExtensions"
```

Comment:
So this breaks "old" configurations?

Shouldn't we have a change log entry for that? Or even better support (time limited) backwards compatibility?

Reply:
see above.